fru.c

```diff
@@ -612,6 +674,67 @@ fru_board_area_t * fru_board_info(const fru_exploded_board_t *board) ///< [in] E
 	return out;
 }
 
+bool fru_decode_board_info(
+    const fru_board_area_t *area, //< [in] encoded board
+    fru_exploded_board_t *board_out //< [out]
+)
+{
+    uint8_t typelen;
+
+	const uint8_t *curr_data = area->data;
```

Comment:
Indentation? Please use tabs for indentation in your code. Spaces may be used for alignment within one indentation level.